<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tke_Dialog_editOperaciiStrings</class>
 <widget class="QDialog" name="tke_Dialog_editOperaciiStrings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>576</width>
    <height>305</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Стрічки нарахування поточного абонента</string>
  </property>
  <property name="windowIcon">
   <iconset resource="tke/tke_rc">
    <normaloff>:/pictures/pictures/cube.png</normaloff>:/pictures/pictures/cube.png</iconset>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="margin">
    <number>2</number>
   </property>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <property name="leftMargin">
      <number>4</number>
     </property>
     <item>
      <widget class="QLabel" name="label_potochnaData">
       <property name="text">
        <string>Поточна дата</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dateEdit_potochnaData">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="correctionMode">
        <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
       </property>
       <property name="maximumDate">
        <date>
         <year>2100</year>
         <month>12</month>
         <day>31</day>
        </date>
       </property>
       <property name="minimumDate">
        <date>
         <year>2005</year>
         <month>1</month>
         <day>1</day>
        </date>
       </property>
       <property name="displayFormat">
        <string>MM.yyyy</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_allColumns">
       <property name="text">
        <string>Усі поля</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_operaciiStrings">
     <property name="title">
      <string>Стрічки операцій поточного абонента</string>
     </property>
     <layout class="QVBoxLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QTableWidget" name="tableWidget_narahStrings">
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <column>
         <property name="text">
          <string>id</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Вх.сал.</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Нарах.ЦО</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Піл.ЦО</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Кат.піл.ЦО</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Пільговики ЦО</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Пільгова площа</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Нарах.ГВ</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Піл.ГВ</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Кат.піл.ГВ</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Пільговики ГВ</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Пільговий об'єм</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Нарах.СТ</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Перерах.РІК</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Перерах.МІС.</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Початк.ЦО</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Кінц.ЦО</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Початк.ГВ</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Кінц.ГВ</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Статус</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton_addOperaciiString">
          <property name="text">
           <string>Додати стрічку</string>
          </property>
          <property name="icon">
           <iconset resource="tke/tke_rc">
            <normaloff>:/pictures/pictures/greplus.png</normaloff>:/pictures/pictures/greplus.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>12</width>
            <height>12</height>
           </size>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextBesideIcon</enum>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton_delOperaciiString">
          <property name="text">
           <string>Видалити стрічку</string>
          </property>
          <property name="icon">
           <iconset resource="tke/tke_rc">
            <normaloff>:/pictures/pictures/redminus.png</normaloff>:/pictures/pictures/redminus.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>12</width>
            <height>12</height>
           </size>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextBesideIcon</enum>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_closeDialog">
       <property name="text">
        <string>Закрити</string>
       </property>
       <property name="icon">
        <iconset resource="tke/tke_rc">
         <normaloff>:/pictures/pictures/button_cancel.png</normaloff>:/pictures/pictures/button_cancel.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>12</width>
         <height>12</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="tke/tke_rc"/>
 </resources>
 <connections/>
</ui>
